zombienet/tests/0007-full_node_warp_sync.toml

```diff
@@ -0,0 +1,77 @@
+[relaychain]
+default_image = "{{RELAY_IMAGE}}"
+default_command = "polkadot"
+default_args = [ "-lparachain=debug" ]
+default_db_snapshot = "https://github.com/samelamin/cumulus/blob/chain-snapshots/docker/snapshots/relaychain-1964f8b557f10085cdc18f4105ad0bbb3df4c4c6.tgz?raw=true"
+chain = "rococo-local"
```

Comment:
We should add the exact relay chain spec that was used to generate the db here again. If this causes problems, please open a zombienet issue.